Public/Src/Sandbox/MacOs/Sandbox/Src/Kauth/AccessHandler.cpp

```diff
@@ -205,6 +205,22 @@ static bool VNodeMatchesPath(vnode_t vp, vfs_context_t ctx, const char *path)
     return result;
 }
 
+const char* AccessHandler::IgnoreCatalinaDataPartitionPrefix(const char* path)
+{
+    if (!sandbox_->GetConfig().enableCatalinaDataPartitionFiltering)
+    {
+        return path;
+    }
+
+    const char *marker = path;
+    if (strprefix(marker, "/System/Volumes/Data/"))
+    {
+        marker = marker + strlen("/System/Volumes/Data");
```

Comment:
How many times is this line going to execute per single build?  If more than once (which seems to be the case), extract `strlen("/System/Volums/Data")` into a constant.

Reply:
Extracted it into a const, much cleaner - good point.